plugins/autowarmth.koplugin/main.lua

```diff
@@ -581,38 +604,86 @@ function AutoWarmth:getSubMenuItems()
             text = Device:hasNaturalLight() and _("Warmth and night mode settings") or _("Night mode settings"),
             sub_item_table = self:getWarmthMenu(),
         },
-        {
-            checked_func = function()
-                return self.fl_off_during_day
-            end,
-            text = _("Frontlight off during the day"),
-            callback = function(touchmenu_instance)
+        self:getFlOffDuringDayMenu(),
+        self:getTimesMenu(_("Currently active parameters")),
+        self:getTimesMenu(_("Sun position information for"), true, activate_sun),
+        self:getTimesMenu(_("Fixed schedule information"), false, activate_schedule),
+    }
+end
+
+function AutoWarmth:getFlOffDuringDayMenu()
+    return {
+        checked_func = function()
+            return self.fl_off_during_day
+        end,
+        text_func = function()
+            if self.fl_off_during_day and  self.fl_off_during_day_offset_s ~= 0 and not self.easy_mode then
+                return T(_("Frontlight off during day: %1 min offset"), math.floor(self.fl_off_during_day_offset_s/60))
+            else
+                return _("Frontlight off during day")
+            end
+        end,
+        callback = function(touchmenu_instance)
+            if self.easy_mode then
                 self.fl_off_during_day = not self.fl_off_during_day
                 G_reader_settings:saveSetting("autowarmth_fl_off_during_day", self.fl_off_during_day)
+            else
+                -- hard limit offset to 15 min after sunset and 30 mins before sunset; sunrise vice versa
+                UIManager:show(SpinWidget:new{
+                    title_text = _("Offset for frontlight off"),
+                    info_text = _[[This is the time before sunset (after sunrise), when the frontlight turns off.
```

Comment:
Is this saying "This is the first time… that the frontlight has turned off"?
```suggestion
                    info_text = _[[This is the time before sunset (after sunrise), when the frontlight turns off.
```

Reply:
I want to say: Frontlight gets turned off from sunsrise to sunset. With this setting the off-time can be from sunrise plus offset to sunset minus offset.

Reply:
Like this?
```suggestion
                    info_text = _[[The frontlight will be turned off at this time before sunset (after sunrise).
```